src/mixins/SizeMixin.js

```diff
@@ -0,0 +1,26 @@
+const sizes = {
+  small: 32,
+  medium: 48,
+  large: 64,
+}
+
+export const SizeMixin = {
+  props: {
+    size: {
+      type: [String, Number],
+      default: 'medium',
+      validator: size => {
+        return typeof size === 'string' || typeof size === 'number'
+      },
+    },
+  },
+
+  methods: {
+    getPropSize (property) {
+      if (typeof this.size === 'string') {
+        return { [property]: this.size in sizes ? `${sizes[this.size]}px` : this.size }
```

Comment:
Wouldn't it make more sense to move property to component?

I.e.
someProperty: getPropSize()
instead of
...getPropSize('someProperty')

Reply:
fixed